src/js/wrappers/ListFilterFormWrapper.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+const ListFilterFormWrapper = ({ children }) => (
+  <div className="d-flex flex-column list-page-filters">
```

Comment:
I was thinking for a while about those wrappers that you created. In the beginning, I loved the idea, but now I am a little bit confused about whether "common styles" should be created using wrappers. I think each of those list pages, filters, etc. should have the same className, and we should have those styles in something like a "common" or "global" scss file.

Reply:
The purpose of the wrappers were to reduce the redundancy of the code for an eventual future refactor. If we ever decided to add/remove/rename a className of those divs, we'd have to do it only in a wrapper, not in every `*List.jsx` file.

Reply:
Another way to achieve the same result would be to store the classNames in a const. I went this way hoping you all would like the idea.